pkg/storage/export/export/export_test.go

```diff
@@ -969,6 +969,8 @@ var _ = Describe("Export controller", func() {
 		})
 		err = controller.createCertSecret(testVMExport, testExportPod)
 		Expect(err).ToNot(HaveOccurred())
+		Expect(controller.Recorder.(*record.FakeRecorder).Events).To(HaveLen(1))
```

Comment:
The line below convers this or not?

Reply:
Not 100% sure if ExpectEvent properly handles if there are no events. So just being defensive.